tap-setup-scripts/src/cloud-init.sh

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env bash
+
+set -ex
+
+arch=$(dpkg --print-architecture)
+user=ubuntu
```

Comment:
I didn't change anything in the scripts (except the bare minimum to make the split between inline'd user-data and externalized cloud-init script happen).
I think setting it once and reusing `$user` is fine, makes it easy enough to switch in case we'd ever need to.